cli/cmd/testdata/routes_one_output.golden

```diff
@@ -1,5 +1,5 @@
-ROUTE       SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99    TLS
-/a          100.00%   1.5rps         123ms         123ms         123ms   100%
-/b          100.00%   1.0rps         123ms         123ms         123ms   100%
-/c            0.00%   0.0rps         123ms         123ms         123ms     0%
-[UNKNOWN]   100.00%   0.5rps         123ms         123ms         123ms   100%
+ROUTE                             SERVICE   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99    TLS
+/a          foo.default.svc.cluster.local   100.00%   1.5rps         123ms         123ms         123ms   100%
+/b          foo.default.svc.cluster.local   100.00%   1.0rps         123ms         123ms         123ms   100%
+/c          foo.default.svc.cluster.local     0.00%   0.0rps         123ms         123ms         123ms     0%
+[UNKNOWN]   foo.default.svc.cluster.local   100.00%   0.5rps         123ms         123ms         123ms   100%
```

Comment:
👍 to @grampelberg comment about displaying service names instead of authorities here.

Reply:
On second thought, I think the second column should reflect the type of the `--to` flag, if you make the flag changes that I suggested above. So `linkerd routes deploy/webapp --to svc` would display either `authors` or `books` in the second column. But `linkerd routes deploy/webapp --to au` would display the full authorities for those services instead.

Reply:
@klingerf so we'd change the column title? I like the idea of making it specific to the view that you're requesting. I assume that `--to svc` is the default?

Reply:
Right, we'd need to change the column title, but services could be the default.

Reply:
I've implemented this as we discussed: 
When the `--to` flag is a service or when there is no `--to` flag and the target is a service, then we change the name from AUTHORITY to SERVICE and parse out the service name.

The result is less useful than I had realized because in all of these cases, you specified the service name in the command line.  Therefore the SERVICE column values will all be the same: the value you specified.